### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by SyafaHadyan** [d4rken-org/sdmaid-se#710] [General]
# Feature Request

|Feature|Information|
|:---|:---|
|**Statistics**|Detailed statistics (app usage, total storage cleaned, etc.)|
|**CSV or PDF Export**|Option to export history|
|**Settings Backup or Sync**|Option to back up settings and or sync settings between devices|
|**Tasker Integration**|Integrate with the Tasker app|
|**Result notification**|A notification that contain result after cleaning|

**Comment by d4rken** [comment]
One feature request per ticket. I think for some of your ideas, there is already an open ticket.
